st2common/st2common/persistence/keyvalue.py

```diff
@@ -19,7 +19,7 @@
 from st2common.constants.triggers import KEY_VALUE_PAIR_UPDATE_TRIGGER
 from st2common.constants.triggers import KEY_VALUE_PAIR_VALUE_CHANGE_TRIGGER
 from st2common.constants.triggers import KEY_VALUE_PAIR_DELETE_TRIGGER
-from st2common.exceptions.db import StackStormDBObjectNotFoundError
+from st2common.exceptions.db import StackStormDBKeyNotFoundError
```

Comment:
I think I can see why the unit-tests are failing. Although we've changed the catch to be for the DataStoreKeyNotFoundError we are raising the StackStormDBKeyNotFoundError.

Can we swap from StackStormDBKeyNotFoundError to st2common.exceptions.keyvalue.DataStoreKeyNotFoundError.